src/test/routes/postcode-lookup.ts

```diff
@@ -0,0 +1,36 @@
+import * as request from 'supertest'
+import { expect } from 'chai'
+import { app } from 'main/app'
+import { Paths } from 'paths'
+import 'test/routes/expectations'
+import * as mock from 'nock'
+import { mockPostcodeLookupResponse } from '../data/entity/mockPostcodeLookupResponse'
+
+describe('PostCode Lookup', () => {
+
+  const mockPostcodeServer = 'https://api.ordnancesurvey.co.uk'
+  const mockPostcodePath = /\/places\/v1\/addresses\/postcode\?.+/
+
+  it('should return correct address when postCode lookup ', async () => {
```

Comment:
Looks like you didn't finish the sentence?

Reply:
haha true, let me change that :)